apps/src/applab/designMode.js

```diff
@@ -394,8 +394,11 @@ designMode.parseFromLevelHtml = function(rootEl, allowDragging) {
   }
 
   children.each(function () {
-    elementLibrary.onDeserialize($(this)[0]);
+    elementLibrary.onDeserialize($(this)[0], designMode.onPropertyChange.bind(this));
   });
+  children.children().each(function() {
+    elementLibrary.onDeserialize($(this)[0], designMode.onPropertyChange.bind(this));
+  })
```

Comment:
I wonder if this part actually belongs in elementLibrary.onDeserialize - i.e. I cal onDeserialize(element), and it takes care of deserializing the element and the element's children.